src/Sarif/NotYetAutogenerated/SarifLogEqualityComparer.cs

```diff
@@ -0,0 +1,95 @@
+// Copyright (c) Microsoft.  All Rights Reserved.
+// Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.CodeDom.Compiler;
+using System.Collections.Generic;
+
+namespace Microsoft.CodeAnalysis.Sarif
+{
+    /// <summary>
+    /// Defines methods to support the comparison of objects of type SarifLog for equality.
+    /// </summary>
+    [GeneratedCode("Microsoft.Json.Schema.ToDotNet", "0.20.0.0")]
+    public sealed class SarifLogEqualityComparer : IEqualityComparer<SarifLog>
+    {
+        public bool Equals(SarifLog left, SarifLog right)
+        {
+            if (ReferenceEquals(left, right))
+            {
+                return true;
+            }
+
+            if (ReferenceEquals(left, null) || ReferenceEquals(right, null))
+            {
+                return false;
+            }
+
+            var runEqualityComparer = new RunEqualityComparer();
+            if (left.SchemaUri != right.SchemaUri)
```

Comment:
Still in NotYetAutoGenerated because I don't yet have "PropertyNameHint" that will map the JSON property `"$schema"` to the .NET property `SchemaUri`.